mp/src/game/shared/momentum/mom_gamemovement.cpp

```diff
@@ -381,9 +383,7 @@ void CMomentumGameMovement::WalkMove()
 
 void CMomentumGameMovement::StepMove(Vector &vecDestination, trace_t &trace)
 {
-#ifdef USE_NEW_RNGFIX
-    if (g_pGameModeSystem->GameModeIs(GAMEMODE_AHOP))
-#endif
```

Comment:
We still want this gamemode check.

Reply:
I assumed that we'd want to test out any RNGFix changes on all game modes, unless the intention is to _never_ use RNGFix in ahop.

Reply:
Correct, it was messing up ahop. So if it's enabled it should check if it's ahop and also early return.